core/app/models/spree/customer_return.rb

```diff
@@ -1,5 +1,6 @@
 module Spree
   class CustomerReturn < Spree::Base
+    include Spree::Core::NumberGenerator.new(prefix: 'CR', length: 9)
```

Comment:
Then wouldn't this remove desired behavior.  For instance not everyone wants their prefix to be `CR` so how would they change it to just `C` in the customer_return_decorator.rb?